clio-integration-test/src/main/python/clio_test.py

```diff
@@ -29,7 +29,7 @@ def test_wait_for_clio():
             js = r.json()
             clio_status = js['clio']
             search_status = js['search']
-            if clio_status == 'Started' and search_status == 'OK':
+            if clio_status == 'Started' : # and search_status == 'OK':
```

Comment:
Is OK no longer expected? Or is this a temporary change that will be reverted when the version number changes are reverted?